app/views/account/webhooks/outgoing/endpoints/_form.html.erb

```diff
@@ -4,6 +4,8 @@
   <% with_field_settings form: form do %>
     <%= render 'shared/fields/text_field', method: :name, options: {autofocus: true} %>
     <%= render 'shared/fields/text_field', method: :url %>
+    <%= render 'shared/fields/super_select', method: :scaffolding_absolutely_abstract_creative_concept_id,
+    choices: [['None', '']] + @endpoint.creative_concepts.map { |event_type| [event_type.name, event_type.id] }, other_options: {search: true} %>
```

Comment:
Can we replace `[['None', '']] +` with `allow_blank: true`? Does that work with `super_select`? /cc @gazayas @pascallaliberte

Reply:
@andrewculver It looks like we can remove `[['None', '']]` and just use the [allowClear](https://select2.org/placeholders) option along with the `placeholder` option Select2 has available.

You can add the options in the stimulus controller [here](https://github.com/bullet-train-co/bullet_train-fields/blob/064658a5acd32ffab38e7ea1b21d80afff32e631/app/javascript/controllers/fields/super_select_controller.js#L63) (I ended up adding it a little further below as `options.allowClear = true;` and it worked fine), and this will pass an empty String to the `params` in the controller, just like with `[['None'], ['']]`

Concerning the Creative Concept's <b>validation</b> though, `allow_blank: true` isn't working for now because the empty string has no ID, just resulting in a `nil` record. For that reason I think the `optional: true` line that was added in this PR in the `EndpointSupport` module might be a viable option. That is if we're ok with the Creative Concept being saved to the database as `nil`.